src/components/EDraggable.vue

```diff
@@ -90,34 +88,63 @@ onMounted(() => {
 
   // window.addEventListener("resize", calculateCoordinates);
 });
+
+function findCoordinates(el: Element, done: () => void) {
+  const $draggableDocked = document.querySelector(
+    `.EDraggablesDock .EDraggableTitlebar[data-id="${draggableId}"]`,
```

Comment:
Ysna rets viis .getBoundingClientRect() k2tte saada, kas seda kuidagi refidega ei saa teha? 

Ma paneks v2hemalt `TODO: Find a way to simplyfy this`, nata yle pea k2ib koik see v2rk siin aga noh, commitime 2ra.